pom.xml

```diff
@@ -50,7 +50,7 @@
         <gridcapa.starter.minio.adapter.version>1.1.0</gridcapa.starter.minio.adapter.version>
         <postgres.jdbc.version>42.3.8</postgres.jdbc.version>
         <logstash.version>7.0.1</logstash.version>
-
+        <commons-lang3.version>3.14.0</commons-lang3.version>
```

Comment:
There is already the version 3.13.0 of commons-lang3 in farao-parent since v1.7.0. It's better to upgrade the farao-parent version and use the same version?

Reply:
I didn't notice there was a common-lang3 dependency in farao-parent. Yes it would be better to update this one rather than duplicating the dependency with a different version here.

Reply:
In fact we maybe don't even need the 3.14 version specifically. Using farao-parent 1.7.0 with common-lang3 3.13.0 seems to work as well. I will update task-manager's pom.xml to use version 1.7.0 of farao-parent.